spec/clipboard.spec.js

```diff
@@ -23,7 +23,69 @@ describe('clipboard.spec: clipboard namespace tests', () => {
             `);
             assert.equal(runner.getOutput(), 'ok');
         });
+        it('throws an error if the parameter is not a string', async () => {
+            let exitCode = runner.run(`
+                try {
+                    await Neutralino.clipboard.writeText(123); // Passing a number instead of a string
+                }
+                catch(err) {
+                    await __close(err.code);
+                }
+            `);
+            assert.equal(runner.getOutput(), 'NE_RT_NATRTER');
+        });
+        
+        it('successfully writes special characters to the clipboard', async () => {
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.writeText('Special characters: @#$%^&*');
+                await __close('ok');
+            `);
+            assert.equal(runner.getOutput(), 'ok');
+        });
+        
+        it('successfully writes a large amount of text to the clipboard', async () => {
+            let largeText = 'A'.repeat(10000000);
+            let exitCode = runner.run(`
+                await Neutralino.clipboard.writeText('${largeText}');
+                await __close('ok');
+            `);
+            assert.equal(runner.getOutput(), 'ok');
+        });
+        it('throws an error if the parameter is null', async () => {
+            let exitCode = runner.run(`
+                try {
+                    await Neutralino.clipboard.writeText(null);
+                }
+                catch(err) {
+                    await __close(err.code);
+                }
+            `);
+            assert.equal(runner.getOutput(), 'NE_RT_NATRTER');
+        });
+        it('throws an error if the parameter is undefined', async () => {
+            let exitCode = runner.run(`
+                try {
+                    await Neutralino.clipboard.writeText(undefined);
+                }
+                catch(err) {
+                    await __close(err.code);
+                }
+            `);
+            assert.equal(runner.getOutput(), 'NE_RT_NATRTER');
+        });
+        it('throws an error if the parameter is an object', async () => {
+            let exitCode = runner.run(`
+                try {
+                    await Neutralino.clipboard.writeText({ key: 'value' });
+                }
+                catch(err) {
+                    await __close(err.code);
+                }
+            `);
+            assert.equal(runner.getOutput(), 'NE_RT_NATRTER');
```

Comment:
Seems like we can merge these tests into `throws an error if the parameter is not a string` case?